locales/fr.js

```diff
@@ -90,41 +90,72 @@ const fr = {
     "Learning catalogue"             : "Catalogue d'apprentissage",
     "Learning tools"                 : "Outils d'apprentissage",
     "Blogs"                          : "Blogues",
+
     // Footer
-    'about_government'                  : 'Au sujet du gouvernement',
-    'about_this_site'                   : 'Au sujet de ce site',
-    'top_of_page'                       : 'Haut de la page',
-    'symbol_of_the_government_of_canada': 'Symbole du gouvernement du Canada',
-    "Date modified"                     : "Date de modification : ",
+    "Date modified": "Date de modification : ",
+
+    "About this site": "À propos de ce site",
+    "Government of Canada": "Gouvernement du Canada",
+
+    "All contacts": "Toutes les coordonnées",
+    "Departments and agencies": "Ministères et organismes",
+    "About government": "À propos du gouvernement",
+    "Themes and topics": "Thèmes et sujets",
+    "Jobs": "Emplois",
+    "Immigration and citizenship": "Immigration et citoyenneté",
+    "Travel and tourism": "Voyage et tourisme",
+    "Business": "Entreprises",
+    "Benefits": "Prestations",
+    "Health": "Santé",
+    "Taxes": "Impôts",
+    "Environment and natural resources": "Environnement et ressources naturelles",
+    "National security and defence": "Sécurité nationale et défense",
+    "Culture, history and sport": "Culture, histoire et sport",
+    "Policing, justice and emergencies": "Services de police, justice et urgences",
+    "Transport and infrastructure": "Transport et infrastructure",
+    "Canada and the world": "Le Canada et le monde",
+    "Money and finance": "Argent et finance",
+    "Science and innovation": "Science et innovation",
+    "Indigenous peoples": "Autochtones",
+    "Veterans and military": "Vétérans et militaires",
+    "Youth": "Jeunesse",
+
+    "Social media": "Médias sociaux",
+    "Mobile applications": "Applications mobiles",
+    "About Canada.ca": "À propos de Canada.caa",
```

Comment:
Juste une petite coquille:
```suggestion
    "About Canada.ca": "À propos de Canada.ca",
```